test/TestMail.hs

```diff
@@ -1,16 +1,19 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Do we still need this? Maybe my misunderstanding, but I thought the switch to quickcheck-instances would have eliminated the need?

Reply:
We now have the orphan instance for `Notmuch.Tag`